next-react-query-tailwind/src/components/UserRepos/UserRepos.data.tsx

```diff
@@ -19,9 +19,9 @@ interface UserReposProps {
 function UserRepos({ username, isOrg = false }: UserReposProps) {
   const { query } = useRouter();
```

Comment:
Looks like we're not using the query param anymore

Reply:
I was mimicking the RxJs pattern on angular-apollo-tailwind and the one you had previously done

Reply:
I'd prefer to keep using the query params actually and maybe upgrade the RxJS one to match this instead of this being changed.